tutorials/developers/dev_nn_datapoint/dev_nn_datapoint.tex

```diff
@@ -0,0 +1,2454 @@
+    'G', g, ...
+    'M_LIST', {'Degree' 'DegreeAv' 'Distance'}, ...
+    'SUB', sub, ...
+    'TARGET_IDS', sub.get('VOI_DICT').get('KEYS')), ...
+    a_BUD.get('G_DICT').get('IT_LIST'), gr.get('SUB_DICT').get('IT_LIST'),...
+    'UniformOutput', false);
+
+% create NNDataPoint_CON_CLA DICT items
+dp_list = IndexedDictionary(...
+        'IT_CLASS', 'NNDataPoint_Measure_REG', ...
+        'IT_LIST', it_list ...
+        );
+
+% create a NNDataset containing the NNDataPoint_Measure_REG DICT
+d = NNDataset( ...
+    'DP_CLASS', 'NNDataPoint_Measure_REG', ...
+    'DP_DICT', dp_list ...
+    );
+
+% Check whether the content of input for a single datapoint matches
+for index = 1:1:gr.get('SUB_DICT').get('LENGTH')
+    individual_input = d.get('DP_DICT').get('IT', index).get('INPUT');
+    known_input = cellfun(@(m) m.get('M'), a_BUD.get('G_DICT').get('IT', index).get('M_DICT').get('IT_LIST'), 'UniformOutput', false);
+
+    assert(isequal(individual_input, known_input), ...
+        [BRAPH2.STR ':NNDataPoint_Measure_REG:' BRAPH2.FAIL_TEST], ...
+        'NNDataPoint_Measure_REG does not construct the dataset correctly. The input value is not derived correctly.' ...
+        )
+end
+
+%%% ¡test!
+%%%% ¡name! ¥\circled{6}\circlednote{6}{tests various kinds of graph measure with the \code{MultiplexWU}.}¥
+Construct the data point with the adjacency matrix derived from its multiplex weighted undirected graph (MultiplexWU) 
+%%%% ¡code!
+% ensure the example data is generated
+if ~isfile([fileparts(which('SubjectCON_FUN_MP')) filesep 'Example data CON_FUN_MP XLS' filesep 'atlas.xlsx'])
+    test_SubjectCON_FUN_MP % create example files
+end
+
+% Load BrainAtlas
+im_ba = ImporterBrainAtlasXLS( ...
+    'FILE', [fileparts(which('SubjectCON_FUN_MP')) filesep 'Example data CON_FUN_MP XLS' filesep 'atlas.xlsx'], ...
+    'WAITBAR', true ...
+    );
+
+ba = im_ba.get('BA');
+
+% Load Groups of SubjectCON
+im_gr = ImporterGroupSubjectCON_XLS( ...
+    'DIRECTORY', [fileparts(which('SubjectCON_FUN_MP')) filesep 'Example data CON_FUN_MP XLS' filesep 'CON_FUN_MP_Group_1_XLS.CON'], ...
+    'BA', ba, ...
+    'WAITBAR', true ...
+    );
+
+gr_CON = im_gr.get('GR');
+
+% Load Groups of SubjectFUN
+im_gr = ImporterGroupSubjectFUN_XLS( ...
+    'DIRECTORY', [fileparts(which('SubjectCON_FUN_MP')) filesep 'Example data CON_FUN_MP XLS' filesep 'CON_FUN_MP_Group_1_XLS.FUN'], ...
+    'BA', ba, ...
+    'WAITBAR', true ...
+    );
+
+gr_FUN = im_gr.get('GR');
+
+% Combine Groups of SubjectCON with Groups of SubjectFUN
+co_gr = CombineGroups_CON_FUN_MP( ...
+    'GR_CON', gr_CON, ...
+    'GR_FUN', gr_FUN, ...
+    'WAITBAR', true ...
+    );
+
+gr = co_gr.get('GR_CON_FUN_MP');
+
+% Analysis CON FUN MP WU
+a_WU = AnalyzeEnsemble_CON_FUN_MP_WU( ...
+    'GR', gr ...
+    );
+
+% To be added the multiplex measures
+a_WU.get('MEASUREENSEMBLE', 'Degree').get('M');
+a_WU.get('MEASUREENSEMBLE', 'DegreeAv').get('M');
+a_WU.get('MEASUREENSEMBLE', 'Distance').get('M');
+
```

Comment:
So actually, these measures should be multiplex, but since all measures are not yet in develop.... that's why. So this, I would change, maybe it does not matter much here. But I want to change all the example files in the multiplex pipelines in the code.

In current braph2 we have for WU pipelines:
WeightedMultiplexParticipation
WeightedMultiplexParticipationAv
WeightedEdgeOverlap

In current braph2 we have for BUD/BUT pipelines:
MultiplexParticipation
OverlappingDegreeAv
EdgeOverlap

Reply:
Yes, I agree. But I also think using a multiplex measure is not of the most essence since this DEV demonstrated the functionality (or how to implement this functionality) of utilizing graph such multiplex to create a datapoint. It would be nice in the future revision but for this PR it is ok in my opinion.

Reply:
yes, it is not incorrect, but i will want to change this on the example codes. So maybe is better to remove this example in the tutorial, like we can just do examples with CON, or other single layer. just one example for each type of nndatapoint